app/src/renderer/styles/variables.styl

```diff
@@ -25,6 +25,7 @@ dim = hsl(app-hue, 13%, 50%)
 
 // borders
 bc = hsl(app-hue, 22%, 23%)
+bc-dim = app-fg
```

Comment:
Really like the look of this border color for reducing visual noise.

Reply:
awesome.